src/m365/connection/commands/connection-use.ts

```diff
@@ -0,0 +1,63 @@
+import { Logger } from '../../../cli/Logger.js';
+import auth from '../../../Auth.js';
+import commands from '../commands.js';
+import Command, { CommandError } from '../../../Command.js';
+import GlobalOptions from '../../../GlobalOptions.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  name: string;
+}
+
+class ConnectionUseCommand extends Command {
+  public get name(): string {
+    return commands.USE;
+  }
+
+  public get description(): string {
+    return "When signed in with multiple identities, switch to another connection";
```

Comment:
Since the command can also be used with one connection, let's simplify its name to `Activate the specified Microsoft 365 tenant connection`